exercises/triangle/description.md

```diff
@@ -2,6 +2,16 @@ The program should raise an error if the triangle cannot exist.
 
 ## Hint
 
-The sum of the lengths of any two sides of a triangle always exceeds or 
-is equal to the length of the third side, a principle known as the _triangle
+In mathematics, a degenerate case is a limiting case in which an element of a
+class of objects is qualitatively different from the rest of the class and hence
+belongs to another, usually simpler, class.
+
+1. If the sum of the lengths of any two sides of a triangle is equal to the
+   length of the third side, there exists a degenerate triangle.
+
+2. The sum of the lengths of any two sides of a triangle always exceeds or is
```

Comment:
Currently, there is a sentence about a degenerate case, then a numbered list, and the first item is a degenerate case, but the second isn't, and to me it seemed to make the relationship unclear (why are the two in a numbered list?). What if we start with saying the triangle inequality first, then explain the degenerate case of the inequality, as well as explaining what a degenerate triangle looks like? That would help us to understand what is meant by "qualitatively different from the rest of the class and hence belongs to another, usually simpler, class." by giving an example relevant to the exercise.